index.html

```diff
@@ -4,7 +4,7 @@
   <meta charset="UTF-8">
   <title>Path of Damage</title>
   <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.6.4/angular.min.js"></script>
-  <script src="lz-string.min.js"></script>
+  <script src="angular-lz-string.js"></script>
```

Comment:
I meant using the non-angular library, but it's aright, I have some tinkering to do later